src/types/index.d.ts

```diff
@@ -45,7 +46,6 @@ export interface FileUploadType {
 // FormData is used by json-schema-forms internally to track state of a single form
 export interface FormData {
   schema?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
-  uiSchema?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
I moved this to `FormTemplate` if that's fine

Reply:
actually, this is used by json-schema-forms internally right? So the object is actually there during runtime...

Reply:
if thats the case, probably should just leave it there, so at least we know there is uiSchema there, but don't need to use it i guess.

Reply:
![image](https://user-images.githubusercontent.com/506667/95420524-29f66c00-096e-11eb-9841-7043c70d6ea6.png)

Yeah I did check the docs - it's used internally, but not by `formData`. Rather, it sits on the same level as `formData` in the `Form` props (see [this](https://react-jsonschema-form.readthedocs.io/en/latest/api-reference/form-props/#uischema) and also a visual representation above showing `uiSchema` and `formData` are actually separate from each other).

So it's a bit weird to have `uiSchema` nested within `formData`, but I'm ok to leave it there too.

Reply:
added it back!